docs/site/src/demos/lists/InsetList.js

```diff
@@ -0,0 +1,37 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import {
+  List,
+  ListItem,
+  ListItemText,
+} from 'material-ui/List';
+
+const styleSheet = createStyleSheet('InsetList', (theme) => ({
+  root: {
+    width: '100%',
+    maxWidth: '360px',
+    background: theme.palette.background.paper,
+  },
+}));
+
+function SimpleList(props, context) {
+  const classes = context.styleManager.render(styleSheet);
+  return (
+    <List className={classes.root}>
+      <ListItem button>
+        <ListItemText inset primary="Chelsea Otakan" />
```

Comment:
I would add an icon, so we better understand the real life use case:
![capture d ecran 2016-12-12 a 22 18 17](https://cloud.githubusercontent.com/assets/3165635/21117177/eb0c1334-c0b8-11e6-9281-577c14ca5c4f.png)